### ISSUE TITLE
Error: Cannot read property 'javascript' of undefined

### ISSUE THREAD

**Issue by Alexsey** [wallabyjs/quokka#417]
### Issue description or question

Instead of showing diff, Quokka is showing an Error

### Sample code

```javascript
({x: 1}), {x: 2}
```

### Quokka.js Console Output

```
Quokka #1 (node: v12.10.0)

Cannot read property 'javascript' of undefined
```

### Code editor version

Webstorm v2019.3 EAP#2

### OS name and version

Windows 10 Pro

**Comment by Alexsey** [comment]
Ok, I see that this way I was accidentally setting the inline config

It would be nice if the error message would explain that inline config has been set unproperly, and not been some internal Quokka error

**Comment by smcenlly** [comment]
This case should have worked for you. We've fixed a bug with Quokka inline configuration that was causing this error message. This is available in Quokka for IntelliJ v1.0.199.
